packages/editor/src/editor-styles/transforms/wrap.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @const string IS_ROOT_TAG Regex to check if the selector is a root tag selector.
+ */
+const IS_ROOT_TAG = /^(body|html).*$/;
```

Comment:
To catch also:
```css
body#test {}
body.test {}
```

Maybe something like this?
```js
const IS_ROOT_TAG = /^(body|html)(?=\s|$|\.|\#)/;
```

Reply:
I'm not certain about this, at least not in Gutenberg's use-case because. It doesn't mean anything to do `body.test` or  `body#test` for editor styles.

Reply:
Should we rename `body` in `body[dir='rtl']` though?

Reply:
That's a good question and even if we don't rename it, it won't work anyway. Because there's no `dir` attribute on the wrapper where we apply the styles. I guess this should be addressed separately.

Reply:
Do you know if there's anything in Gutenberg setting `dir`? If there is, renaming it to `[dir='rtl'] .wrapper` (with space) could work?

Reply:
Yes, there's probably a class or attribute on the body like any other WPAdmin page which means we should introduce a custom "transform" to achieve it.